### ISSUE TITLE
No dev dataset

### ISSUE THREAD

**Issue by ECNUHP** [st9007a/ChineseQABot#7]
May i have a question?
Is there no a dev dataset for validate the model in the training?

**Comment by st9007a** [comment]
Yes, I didn’t prepare a validation set for some reasons.

First, the answer generated from my model will be different from the answer of ground truth, even if I feed a question from training set. So, if I use a validation set to calculate the performance of question- answer directly , I believe I will get a bad performance. Maybe calculating the text perplexity is a solution. But I didn’t implement it.
Second, it is the main reason. I think this project is just a little experiment (or toy). I want to use Transformer to train a chatbot. In fact, Transformer is a deep learning model used for machine translation. I am not sure the result if I use it to train a chatbot. So, I am didn’t prepare a validation set. This project is just a try-and-error.

For some reasons, I have no time to maintain it. Maybe I will add validation in the further.
